pkg/route53.go

```diff
@@ -112,7 +112,7 @@ func (e *Route53Exporter) getHostedZonesPerAccountMetrics(client *servicequotas.
 // CollectLoop runs indefinitely to collect the route53 metrics in a cache. Metrics are only written into the cache once all have been collected to ensure that we don't have a partial collect.
 func (e *Route53Exporter) CollectLoop() {
 	route53Svc := route53.New(e.sess)
-	serviceQuotaSvc := servicequotas.New(e.sess)
+	awsclient := awsclient.NewClientFromSession(e.sess)
```

Comment:
I would not name the variable the same than the module being imported. Maybe just `client` ?